test/regression.uts

```diff
@@ -1880,1941 +1880,6 @@ assert(q[IPOption_LSRR].get_current_router() == "1.2.3.4")
-in6_mactoifaceid("FD:00:00:00:00:00", ulbit=1) == 'FF00:00FF:FE00:0000'
-
-= in6_mactoifaceid() conversion function (test 3)
-in6_mactoifaceid("FD:00:00:00:00:00") == 'FF00:00FF:FE00:0000'
-
-= in6_mactoifaceid() conversion function (test 4)
-in6_mactoifaceid("FF:00:00:00:00:00") == 'FD00:00FF:FE00:0000'
-
-= in6_mactoifaceid() conversion function (test 5)
-in6_mactoifaceid("FF:00:00:00:00:00", ulbit=1) == 'FF00:00FF:FE00:0000'
-
-= in6_mactoifaceid() conversion function (test 6)
-in6_mactoifaceid("FF:00:00:00:00:00", ulbit=0) == 'FD00:00FF:FE00:0000'
-
-########### iface ID conversion #################
-
-= in6_mactoifaceid() conversion function (test 1)
-in6_ifaceidtomac(in6_mactoifaceid("FD:00:00:00:00:00", ulbit=0)) == 'ff:00:00:00:00:00'
-
-= in6_mactoifaceid() conversion function (test 2)
-in6_ifaceidtomac(in6_mactoifaceid("FD:00:00:00:00:00", ulbit=1)) == 'fd:00:00:00:00:00'
-
-= in6_mactoifaceid() conversion function (test 3)
-in6_ifaceidtomac(in6_mactoifaceid("FD:00:00:00:00:00")) == 'fd:00:00:00:00:00'
-
-= in6_mactoifaceid() conversion function (test 4)
-in6_ifaceidtomac(in6_mactoifaceid("FF:00:00:00:00:00")) == 'ff:00:00:00:00:00'
-
-= in6_mactoifaceid() conversion function (test 5)
-in6_ifaceidtomac(in6_mactoifaceid("FF:00:00:00:00:00", ulbit=1)) == 'fd:00:00:00:00:00'
-
-= in6_mactoifaceid() conversion function (test 6)
-in6_ifaceidtomac(in6_mactoifaceid("FF:00:00:00:00:00", ulbit=0)) == 'ff:00:00:00:00:00'
-
-
-= in6_addrtomac() conversion function (test 1)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FD:00:00:00:00:00", ulbit=0)) == 'ff:00:00:00:00:00'
-
-= in6_addrtomac() conversion function (test 2)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FD:00:00:00:00:00", ulbit=1)) == 'fd:00:00:00:00:00'
-
-= in6_addrtomac() conversion function (test 3)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FD:00:00:00:00:00")) == 'fd:00:00:00:00:00'
-
-= in6_addrtomac() conversion function (test 4)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FF:00:00:00:00:00")) == 'ff:00:00:00:00:00'
-
-= in6_addrtomac() conversion function (test 5)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FF:00:00:00:00:00", ulbit=1)) == 'fd:00:00:00:00:00'
-
-= in6_addrtomac() conversion function (test 6)
-in6_addrtomac("FE80::" + in6_mactoifaceid("FF:00:00:00:00:00", ulbit=0)) == 'ff:00:00:00:00:00'
-
-########### RFC 3041 related function ###############################
-= Test in6_getRandomizedIfaceId
-
-import socket
-
-for a in six.moves.range(10):
-    s1, s2 = in6_getRandomizedIfaceId('20b:93ff:feeb:2d3')
-    s1, s2
-    tmp = inet_pton(socket.AF_INET6, "::" + s1)[8:]
-    tmp
-    assert (orb(tmp[0]) & 0x04) == 0
-    s1, s2 = in6_getRandomizedIfaceId('20b:93ff:feeb:2d3', previous=s2)
-    s1, s2
-    tmp = inet_pton(socket.AF_INET6, "::" + s1)[8:]
-    assert (orb(tmp[0]) & 0x04) == 0
-
-assert in6_getRandomizedIfaceId('20b:93ff:feeb:2d3', previous='d006:d540:db11:b092') == ('721f:11fa:3743:fc7f', '5946:5272:7fcc:108a')
```

Comment:
You should copy this test again, it has changed since the rebase.

Reply:
Good catch! Sorry about that.